src/main/java/com/a2/backend/entity/User.java

```diff
@@ -31,5 +31,7 @@
 
     @JsonIgnore String confirmationToken;
 
+    String passwordRecoveryToken;
```

Comment:
@JsonIgnore missing

Reply:
For some reason the @JsonIgnore threw an error that I couldn't find the solution to.